app/containers/cpacpy/SchedulerSelector.js

```diff
@@ -268,6 +298,38 @@ class CpacpySchedulerSelector extends Component {
                   </Paper>
                 ))
               }
+              {/*  Add New BackEnds */}
+              <Paper key='connectNewBackend' elevation={3} className={classes.scheduler}>
+                <Grid container spacing={0}>
+                  <Grid item xs={12} className={classes.description}>
+                    <Grid container spacing={1}>
+                      <Grid item xs={3}>
+                        <TextField
+                          label="Unique Name" fullWidth margin="normal" variant="outlined"
+                          name="general.environment.memory"
+                          onChange={this.handleNewName}
+                        />
+                        <Button variant="contained" color="secondary" onClick={this.handleAddBackend}>Add new
+                          backend</Button>
```

Comment:
I believe we are missing a "test" button, to check if it can connect to the beckend.

Reply:
I added a new `test` method in `sagas/cpacpy.js` because the previous `detect` method is too complicated for this usage. I also added a `testingScheduler` state for `cpacpy` to store state changes for the `test` method in sagas.